src/components/NewsModal/TextualContents/model/enum/CategoryType.ts

```diff
@@ -0,0 +1,10 @@
+type CategoryType =
+  | 'NEWS'
+  | 'MISC'
+  | 'TERMS'
+  | 'CONTACT'
+  | 'PRIVACYPOLICY'
+  | 'IMPRINT'
+  | 'LINK_ONLY';
```

Comment:
See the corresponding question in [PR 1060](https://github.com/terrestris/shogun/pull/1060).